.github/workflows/deb-build-bullseye.yml

```
@@ -0,0 +1,89 @@
+name: Debian Package Build
+
+on:
+  pull_request:
+    branches:
+      - bullseye
+  push:
+    branches:
+      - bullseye
+
+env:
+  DEB_BUILD_DOCKER_IMAGE: "pitop/pi-top-os-deb-build"
+  DEB_BUILD_DOCKER_TAG: "bullseye-master"
+  CHANGELOG_AUTHOR_NAME: "pi-top"
+  CHANGELOG_AUTHOR_EMAIL: "deb-maintainers@pi-top.com"
+  REPO: "experimental"
```

```suggestion
  REPO: "bullseye-experimental"
```
we need to create the `bullseye-experimental` repo in packagecloud